CODEOWNERS

```diff
@@ -13,3 +13,5 @@
 /cloudbuild.yaml                @getsentry/releases
 /scripts/bump-version.sh        @getsentry/releases
 /scripts/post-release.sh        @getsentry/releases
+
+/snuba/migrations/snuba_migrations/profiles @getsentry/profiling
```

Comment:
Mind removing this line? The path has changed so it is causing a failure. Though it is our eventual goal that different teams will own separate paths within the Snuba repo we are not quite there yet.